### ISSUE TITLE
[Keyboard Navigation - PowerApps Admin - Project Tab - Context Menu] 'Context menu' button is not accessible via keyboard Navigation Under Project tab. 

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#7132]
<!-- 🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨

We’re here to help, but we need a few things from you! 
User Experience:
If, any control is not accessible by the keyboard, then the users who depends upon keyboard for navigation might get affected and may not be able to use the complete functionality of the application.

MAS Ref:
MAS 2.1.1 - Keyboard (36)

Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.  

### Environment Information 
 Test Environment:
OS version: Version 1803 (OS build 17134. 228)
Browser: Edge, Chrome

- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant) 

### Please provide a reproduction of the bug in a codepen:
 Repro Steps:
1. Navigate to https://admin.powerplatform.microsoft.com/ext/DataIntegration  and login with intusr.
2. Navigate to the Data Integration menu item on the left bar and select it.
3. Navigate on the controls of data integration page thus opened.
4. Navigate to 'Projects' tab and select it. 
5. Navigate with the keyboard using TAB key on the table.
6. Verify whether the Context menu button is accessible via keyboard or not.
 
#### Actual behavior:
When user navigates with the keyboard using TAB key on the table, then the Context menu button is not accessible by the keyboard. Keyboard virtual focus does not moves to the Delete button.  

 #### Expected behavior:
 When user navigates with the keyboard using TAB key on the table, then the Context menu button is not accessible by the keyboard. Keyboard virtual focus does not moves to the Delete button. 

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low)  Blocking
 
Products/sites affected: (if applicable)

**Comment by sonimishra** [comment]
Comment from Product team: 
The details are as follows:
When we use Overflow set within a cell in DetailsList and when we try navigating using keyboard within the row of the detailslist using right arrow, the focus does move to the OverflowSet as tabindex gets set to 0 but it does not show a focus border. 
If you test in scan mode as well, you will see that the narrator announces the information of the menu/OverflowSet button, however, there is no focus border shown. And if you press space/enter, the narrator announces "Item cannot take focus".

**Comment by aneeshack4** [comment]
@sonimishra Please provide a Codepen that repros this bug so that we can understand what about your implementation of OverflowSet as a cell of DetailsList
